Summary:
Internal error processing insert java.lang.AssertionError at org.apache.cassandra.service.StorageProxy.sendMessages(StorageProxy.java:219)
Description:
ERROR [pool-1-thread-137] 2011-01-03 18:22:21,751 Cassandra.java (line 2960) Internal error processing insert
java.lang.AssertionError
at org.apache.cassandra.service.StorageProxy.sendMessages(StorageProxy.java:219)
at org.apache.cassandra.service.StorageProxy.mutate(StorageProxy.java:174)
at org.apache.cassandra.thrift.CassandraServer.doInsert(CassandraServer.java:412)
at org.apache.cassandra.thrift.CassandraServer.insert(CassandraServer.java:349)
at org.apache.cassandra.thrift.Cassandra$Processor$insert.process(Cassandra.java:2952)
at org.apache.cassandra.thrift.Cassandra$Processor.process(Cassandra.java:2555)
at org.apache.cassandra.thrift.CustomTThreadPoolServer$WorkerProcess.run(CustomTThreadPoolServer.java:167)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
at java.lang.Thread.run(Thread.java:636)
Status:
RESOLVED
Priority:
Normal
Resolution:
Fixed
Affects_version:

Fix_version:
0.7.1
Component:
None
Label:
None
Environment:
linux, fedora
Attachment number:
0
Assignee:
T Jake Luciani
Reporter:
Karl Mueller
Create date:
04/Jan/11 05:29
Update date:
16/Apr/19 09:33
Resolved date:
04/Jan/11 21:34
